src/client-agent/main.c

```diff
@@ -141,6 +147,22 @@ int main(int argc, char **argv)
         merror_exit(CLIENT_ERROR);
     }
 
+    /* Check if is valid server addres */
+    while(i < agt->rip_id){
+
+        if ( strcmp(agt->server[i].rip, val_default) != 0
```

Comment:
The expected behavior in `Linux` and `Windows` should be the same. Please don't start the agent if the value is `0.0.0.0`.